Assets/Scripts/Interactables/Doohickey.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Doohickey : Interactable
```

Comment:
this is an example of how we'll create new `Interactables`, extended from the base class